apps/sparo-lib/src/services/GitSparseCheckoutService.ts

```diff
@@ -196,12 +199,14 @@ export class GitSparseCheckoutService {
        */
       switch (checkoutAction) {
         case 'purge':
-        case 'skeleton':
           // re-apply the initial paths for setting up sparse repo state
           this._prepareMonorepoSkeleton({
             restore: checkoutAction === 'purge'
           });
           break;
+        case 'skeleton':
```

Comment:
Seems we don't need to support checkoutAction='skeleton' in this function anymore ?

Maybe we can remove it in further PR.

Reply:
Sparo supports checkout to "skeleton" by specifying "--no-profile". So, we are not losing the functionality. Things could be improved is the code organization. Currently, it's a little bit hard to maintain functions in the `GitSparseCheckoutService` class.

Reply:
I checked in the codebase. It's used in "sparo-ci clone". I can see it's a little bit hard to maintain the functions in `GitSparseCheckoutService` now. It could be improved.

Reply:
Agreed.